web/libs/editor/src/regions/Area.js

```diff
@@ -57,12 +60,19 @@ const Area = types.union(
       const available = Registry.getAvailableAreas(tag.type, sn);
       // union of all available Areas for this Object type
 
+      // @todo dirty hack to distinguish two video types
+      if (tag.type === "video") {
+        if (sn.sequence) return VideoRectangleRegionModel;
```

Comment:
It looks like this condition is fine for drawing a new region but will not work with restoring regions from the previously created annotation.

Reply:
yeah, fixed this, thank you! still feels weird, but current way of detecting region type is definitely not the ideal